packages/core/src/modules/connections/services/ConnectionService.ts

```diff
@@ -213,14 +213,18 @@ export class ConnectionService {
     connectionRecord.assertRole(ConnectionRole.Inviter)
 
     if (!message.connection.didDoc) {
-      throw new AriesFrameworkError('Public DIDs are not supported yet')
+      throw new ConnectionProblemReportError(
+        'Public DIDs are not supported yet',
+        ConnectionProblemReportReason.RequestNotAccepted
+      )
     }
 
     // Create new connection if using a multi use invitation
     if (connectionRecord.multiUseInvitation) {
       if (!routing) {
-        throw new AriesFrameworkError(
-          'Cannot process request for multi-use invitation without routing object. Make sure to call processRequest with the routing parameter provided.'
+        throw new ConnectionProblemReportError(
+          'Cannot process request for multi-use invitation without routing object. Make sure to call processRequest with the routing parameter provided.',
+          ConnectionProblemReportReason.RequestNotAccepted
```

Comment:
This is a framework error so I think we  shouldn't send a problem report for it. Or at least not with this message and this reason